packages/common/src/eips/3198.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "EIP-3198",
+    "number": 3198,
+    "comment": "BASEFEE opcode",
+    "url": "https://eips.ethereum.org/EIPS/eip-3198",
+    "status": "Review",
+    "minimumHardfork": "london",
```

Comment:
Same here, minimum hardfork: `berlin`.

For EIP dependency please use the `requiredEIPs` property instead, see e.g. the `2930.json` definition file with `"requiredEIPs": [2718, 2929]`.

Reply:
Will resolve conversations here when doing the local changes and then later push. I have now added `2930` as the sole required EIP, I guess this should do it with `2930` itself having `2718` and `2929` as EIP requirements, which are then also enforced automatically.